### PR TITLE
Documentation for #8988 Fix octoprint errors when printer is off/disconnected 

### PR DESCRIPTION
**Pull request by w1ll1am23**
**Description:**
Doc for two new config options. 

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8988

### CODE REVIEW

File: source/_components/octoprint.markdown [hunk 1]
```diff
@@ -23,12 +23,14 @@ To get started with the OctoPrint API, please follow the directions on their [si
...
+  bed: false
+  number_of_tools: 1
 ```
 
 Configuration variables:
 
 - **host** (*Required*): IP address or hostname of Octoprint host.
 - **api_key** (*Required*): The retrieved api key.
+- **bed** (*Optional*): If the printer has a heated bed.
+- **number_of_tools** (*Optional*): Number of temperature adjustable tools. i.e. nozzel.
```
Comment (frenck) [inline source/_components/octoprint.markdown]: Spelling error:

`nozzel` -> `nozzle`

File: source/_components/sensor.octoprint.markdown [hunk 2]
```diff
@@ -41,3 +41,4 @@ Configuration variables:
   - **Temperatures**:  Temperatures of all available tools, eg. `print`, `head`, `print bed`, etc. These will be displayed as `tool0`, `tool1`, or `toolN` please refer to your OctoPrint frontend to associate the tool number with an actual device.
   - **Job Percentage**: 
 
+**NOTE:** If you are tracking temperature it is recommened to set `bed` and/or `number_of_tools` in your octoprint config. This will allow the octoprint sensors to load if the printer is offline during Home Assistant start up.
```
Comment (frenck) [inline source/_components/sensor.octoprint.markdown]: Spelling errors:
- `recommened` -> `recommended`
- `start up` -> `startup`

File: source/_components/octoprint.markdown (hunk 1 above)
Comment (w1ll1am23) [inline source/_components/octoprint.markdown]: Fixed, thanks.

File: source/_components/sensor.octoprint.markdown (hunk 2 above)
Comment (w1ll1am23) [inline source/_components/sensor.octoprint.markdown]: Fixed, thanks.

**Pull request merged**